pyproject.toml

```
@@ -0,0 +1,3 @@
+[build-system]
+requires = ["setuptools", "wheel"]
```

It's a good idea to document the lowest supported setuptools here (even though the PEP 517 front-end will download the most recent release).
```suggestion
requires = ["setuptools >= 39.2.0", "wheel"]
```